### DISCUSSION TITLE
gnuradio 3.9 does not include gr-soapy

### DISCUSSION THREAD

**Discussion by NeuroForLunch** [Homebrew/discussions#1696] [Writing Formulae/Casks] [answered]
brew install gnuradio installs 3.9 without the core component of gr-soapy. This is because gnuradio first checks for the existance of Soapy before installing gr-soapy. The pre-built binaries were built on servers that don't have Soapy and even if I have Soapy installed, it is too late. Can gnuradio be rebuilt with https://github.com/pothosware/SoapyRTL and https://github.com/pothosware/SoapyRTLSDR pre-installed so gr-soapy gets installed? Or use an override of some kind?

**Comment by carlocab** [comment, marked as answer]
You're welcome to open a PR that adds gr-soapy support to gnuradio for review. These might help: https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request, https://docs.brew.sh/Formula-Cookbook

Alternatively, you can maintain a version of the gnuradio formula with gr-soapy support in your own tap. Here are some docs to help you get started:

https://docs.brew.sh/Taps
https://docs.brew.sh/Interesting-Taps-and-Forks
https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap
https://brew.sh/2020/11/18/homebrew-tap-with-bottles-uploaded-to-github-releases/
